webui/react/src/hooks/useRestApi.test.ts

```diff
@@ -1,6 +1,6 @@
 import * as io from 'io-ts';
 
-import { applyMappers } from './useRestApi';
+import { applyMappers } from 'utils/data';
```

Comment:
nice, good call

Reply:
the motivation was for the jest tests to not have to import useRestApi module to test. which was breaking when importing monaco